data/config-schema/src/main/resources/net/corda/schema/configuration/crypto/1.0/corda.crypto.json

```diff
@@ -72,32 +72,32 @@
       }
     },
     "hsm": {
-      "description": "Settings and capabilities for the HSM",
+      "description": "Settings and capabilities for the HSM.",
       "type": "object",
       "default": {},
       "properties": {
         "retrying": {
-          "description": "Retry settings for the HSM",
+          "description": "Retry settings for the HSM.",
           "type": "object",
           "default": {},
           "properties": {
             "maxAttempts": {
-              "description": "Maximum number of attempts",
+              "description": "The maximum number of attempts.",
```

Comment:
Should we give more info here? Attempts to what?

Reply:
I think this is unused, so it's hard to add more detail.

Reply:
Ok. Let's leave as is. Can we merge the PR?